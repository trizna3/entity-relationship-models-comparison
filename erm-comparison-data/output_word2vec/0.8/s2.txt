osoby-EMPTY
adresy-EMPTY
zmluvy-zmluva
zmluvy na poistenie nehnutelnosti-EMPTY
zmluvy na zivotne poistenie-EMPTY
konta-konta
typy nehnutelnosti-EMPTY
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-odkedy-zmluva
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-dokedy-zmluva
