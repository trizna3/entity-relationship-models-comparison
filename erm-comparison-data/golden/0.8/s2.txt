konta-konta
zmluvy-zmluva
zmluvy na zivotne poistenie-EMPTY
zmluvy na poistenie nehnutelnosti-EMPTY
osoby-EMPTY
adresy-EMPTY
typy nehnutelnosti-EMPTY
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-odkedy-zmluva
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-dokedy-zmluva